environments/se/core/src/main/java/org/jboss/weld/environment/se/DefaultExceptionHandler.java

```diff
@@ -0,0 +1,21 @@
+package org.jboss.weld.environment.se;
```

Comment:
Missing license header...

Reply:
Also in the ``ExceptionHandler`` interface.